components/navbar/Navbar.js

```diff
@@ -116,7 +117,18 @@ export default function Navbar() {
       {session && (
         <>
           <NavLink
-            item={{ name: "Account", url: "/account/onboarding" }}
+            item={{
+              name: (
+                <Image
+                  className="flex-none rounded-full mx-2 bg-primary-low"
+                  width={42}
+                  height={42}
+                  src={`https://github.com/${session.username}.png`}
+                  alt="description of image"
```

Comment:
Would it be better if we add a bit more decriptive and accurate value to `alt` attribute in order to improve accessibility?
The following resource could be helpful:
[Resource for Image Alt text best practises](https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices#:~:text=The%20alt%20text%20needs%20to,for%20those%20using%20screen%20readers.)

Reply:
Sure, I'll change it to a more descriptive text